graph/src/data/subgraph/mod.rs

```diff
@@ -52,6 +52,8 @@ pub const SPEC_VERSION_0_0_4: Version = Version::new(0, 0, 4);
 
 pub const MIN_SPEC_VERSION: Version = Version::new(0, 0, 2);
 
+pub const SPEC_VERSION_0_0_1: Version = Version::new(0, 0, 1);
```

Comment:
The spec version should be general, rather than chain-specific.

Reply:
I thought the spec was defining some kind of versioning for the AssemblyScript API, isn't the case?

Reply:
That's the `apiVersion`, which is within the `dataSource`. Whether different chains will have different sequences of apiVersions is up for debate. Ping @otaviopace and @azf20 so everyone is aware of the open question. In theory the apiVersion could be independent, in practice a lot of the runtime structure being versioned is shared among chains, and it might be just confusing to keep track of a bunch of independent versions.

Reply:
We decided that for now, the apiVersion versioning will be the same for all blockchains.